### ISSUE TITLE
Pre-trained Pascal VOC2007 model

### ISSUE THREAD

**Issue by Knight13** [fizyr/keras-retinanet#279]
Is there any active link to the pre-trained model on the Pascal VOC2007 dataset? I've evaluated the model pre-trained on MS COCO on my own test set and am trying to compare the results with the model pre-trained on the MS COCO dataset. 

I'd be grateful If anyone has any information on the same. It'd save me a lot of time and resources.

Thanks!

**Comment by hgaiser** [comment]
It should be relatively easy to make a model for VOC (using the `--weights` argument, most of the weights can be copied to a VOC model). Most of the focus from my side has been COCO and I don't intend to add a VOC model anytime soon. This repository is under development, meaning the architecture changes from time to time, meaning a new model has to be created. I prefer to keep that to one dataset, which would be COCO considering the research paper was also focusing on COCO.
